src/types/v2.ts

```diff
@@ -94,18 +153,25 @@ export interface OpenNodeWithdrawal {
   type: string;
   amount: number;
   fee: number;
-  fiat_value: string;
+  fiat_value: number;
   custom_id: string;
   callback_url: string;
   processed_at: number;
-  status: string;
+  conversion?: {
+    from: string;
+    from_amount: number;
+    to: string;
+    to_amount: number;
+  };
+  expiry?: number;
+  status: "completed" | "initial" | "pending" | "failed";
```

Comment:
initial is not public facing

confirmed is also a state (legacy compatible), completed only used on "wires" public faced-responses

Reply:
Will revert to string to accommodate all cases